raiden-ts/src/channels/utils.ts

```diff
@@ -313,3 +313,14 @@ export function approveIfNeeded$(
   );
 }
 /* eslint-enable jsdoc/valid-types */
+
+/**
+ * Predicate to that fiters erros for retrieable network problems. To be used with `retryAsync$`.
+ *
+ * @param error - Error in question
+ * @returns `False`, if there was a retrieable network error.
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function networkErrorRetryPredicate(error: any, {}): boolean {
+  return !error?.includes(`invalid response`);
```

Comment:
Error is most likely an `Error` instance, so you should use something like:
```suggestion
  return !(typeof error?.message === 'string' && error.message.includes(`invalid response`));
```